pants.toml

```diff
@@ -129,7 +129,7 @@ args = [ "--suppress-no-test-exit-code" ]
 [mypy]
 version = "mypy==0.812"
 config = "src/python/mypy.ini"
-args = [ "--show-error-codes" ] #, "--verbose" ]
+args = [ "--show-error-codes", "--namespace-packages" ] #, "--verbose" ]
```

Comment:
I'd honestly add a comment with a link to our slack convo about why this was needed (or put that in a ticket or something)

Reply:
added comment explaining this flag